pkg/controller/storagecluster/initialization_reconciler_test.go

```diff
@@ -450,6 +451,12 @@ func assertExpectedExternalResources(t assert.TestingT, reconciler ReconcileStor
 			for param, value := range expected.Data {
 				assert.Equal(t, value, actual.Parameters[param])
 			}
+			if actual.Name == "ocsinit-ceph-rgw" {
```

Comment:
I would think you should check for the generated name rather than a hard-coded name, no? Also, please add a comment explaining why this is a special case.

Reply:
I can't really check for the generated name since the blob contains all the SC's names so I need to hardcode which one I want to address.